src/components/dialogs/parameters/sensi/utils.ts

```diff
@@ -282,6 +339,42 @@ interface INewParamsInjectionsSet {
     }>;
 }
 
+interface IRowNewParams {
+    [MONITORED_BRANCHES]: Array<{
+        [ID]: string;
+    }>;
+    [INJECTIONS]: Array<{
+        [ID]: string;
+    }>;
+    [HVDC_LINES]: Array<{
+        [ID]: string;
+    }>;
+    [PSTS]: Array<{
+        [ID]: string;
+    }>;
+    [CONTINGENCIES]: Array<{
+        [ID]: string;
+    }>;
+}
+
+export const getGenericRowNewParams = (newRowParams: IRowNewParams) => {
+    return {
+        [MONITORED_BRANCHES]: newRowParams[MONITORED_BRANCHES].map(
+            (container) => container[ID]
+        ),
+        [INJECTIONS]: newRowParams[INJECTIONS]?.map(
+            (container) => container[ID]
+        ),
+        [HVDC_LINES]: newRowParams[HVDC_LINES]?.map(
+            (container) => container[ID]
+        ),
+        [PSTS]: newRowParams[PSTS]?.map((container) => container[ID]),
+        [CONTINGENCIES]: newRowParams[CONTINGENCIES].map(
```

Comment:
newRowParams[CONTINGENCIES]?.map
because contingencies can be empty

Reply:
done